src/Raven.Client/Documents/Operations/TimeSeries/GetMultipleTimeSeriesOperation.cs

```diff
@@ -0,0 +1,122 @@
+            : this(docId, start, pageSize)
+        {
+            _ranges = new List<TimeSeriesRange>
+            {
+                new TimeSeriesRange
+                {
+                    Name = timeseries,
+                    From = from,
+                    To = to
+                }
+            };
+        }
+
+        public GetMultipleTimeSeriesOperation(string docId, IEnumerable<TimeSeriesRange> ranges, int start = 0, int pageSize = int.MaxValue)
+            : this(docId, start, pageSize)
+        {
+            _ranges = ranges ?? throw new ArgumentNullException(nameof(ranges));
+        }
+
+        private GetMultipleTimeSeriesOperation(string docId, int start, int pageSize)
+        {
+            if (string.IsNullOrEmpty(docId))
+                throw new ArgumentNullException(nameof(docId));
+
+            _docId = docId;
+            _start = start;
+            _pageSize = pageSize;
+        }
+
+        public RavenCommand<TimeSeriesDetails> GetCommand(IDocumentStore store, DocumentConventions conventions, JsonOperationContext context, HttpCache cache)
+        {
+            return new GetMultipleTimeSeriesCommand(_docId, _ranges, _start, _pageSize);
+        }
+
+        internal class GetMultipleTimeSeriesCommand : RavenCommand<TimeSeriesDetails>
+        {
+            private readonly string _docId;
+            private readonly IEnumerable<TimeSeriesRange> _ranges;
+            private readonly int _start;
+            private readonly int _pageSize;
+
+            public GetMultipleTimeSeriesCommand(string docId, IEnumerable<TimeSeriesRange> ranges, int start, int pageSize)
+            {
+                _docId = docId ?? throw new ArgumentNullException(nameof(docId));
+                _ranges = ranges;
+                _start = start;
+                _pageSize = pageSize;
+            }
+
+            public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
+            {
+                var pathBuilder = new StringBuilder(node.Url);
+                pathBuilder.Append("/databases/")
+                    .Append(node.Database)
+                    .Append("/timeseries/ranges")
+                    .Append("?docId=")
+                    .Append(Uri.EscapeDataString(_docId));
+
+                if (_start > 0)
+                {
+                    pathBuilder.Append("&start=")
+                        .Append(_start);
+                }
+
+                if (_pageSize < int.MaxValue)
+                {
+                    pathBuilder.Append("&pageSize=")
+                        .Append(_pageSize);
+                }
+
+                foreach (var range in _ranges)
+                {
+                    pathBuilder.Append("&name=")
+                        .Append(range.Name)
+                        .Append("&from=")
+                        .Append(range.From.GetDefaultRavenFormat())
```

Comment:
We need to make sure this in UTC.
(We can do that after https://github.com/ravendb/ravendb/pull/10517 is merged)